articles/quickstart/backend/java-spring-security5/interactive.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Spring Boot API
+description: Secure your API using Spring Security 5 and Auth0
+interactive: true
+alias:
+  - spring security
+  - spring
+topics:
+  - quickstart
+  - backend
+  - spring
+files:
+  - files/application
+  - files/audience-validator
+  - files/security-config
+  - files/message
+  - files/api-controller
+github:
+  path: 01-Authorization-MVC
+---
+
+# Secure your API using Spring Security 5 and Auth0
+
+Auth0 allows you to quickly add authorization to your application. This guide demonstrates how to integrate Auth0 with any new or existing Spring Boot application.
+
+If you haven't created an API in your Auth0 dashboard yet, you can use the interactive selector to create a new Auth0 API or select an existing API that represents the project you want to integrate with.
+
+Alternatively, you can read [our getting started guide](get-started/auth0-overview/set-up-apis) that helps you set up your first API through the Auth0 dashboard.
+
+Every API in Auth0 is configured using an API Identifier that your application code will use as the Audience to validate the access token.
+
+<%= include('../../../_includes/_api_auth_intro') %>
+
+## Define permissions
+<%= include('../_includes/_api_scopes_access_resources') %>
+
+## Configure the sample project {{{ data-action=code data-code="application.yml#1:8" }}}
+
+The sample project uses a `/src/main/resources/application.yml` file, which configures it to use the correct Auth0 **domain** and **API Identifier** for your API. If you download the code from this page it will be automatically configured. If you clone the example from GitHub, you will need to fill it in yourself.
```

Comment:
I am wondering if leaving something that references "download" would make sense? Because it actually is about the example that you can download, would it confuse people if we remove the "download" part in the sense that they might not know what example we are referring to?

Reply:
Updated.